web-client/public/js/update-app.js

```diff
@@ -372,8 +372,11 @@ const enableNodeColoringUI = function () {
 const loadExpressionDatabase = function (isTopDataset) {
     startLoadingIcon();
     queryExpressionDatabase({
+        type:"ExpressionData",
         dataset: isTopDataset ? grnState.nodeColoring.topDataset : grnState.nodeColoring.bottomDataset,
-        genes : grnState.workbook.genes
+        genes : grnState.workbook.genes.map(x => {
+            return x.name;
+        }).join(",")
```

Comment:
Might be worth a separate issue to see if we can configure the linter to accept the one-line version of this

Reply:
I fixed that! And made them all one liners again